jep/0000/README.adoc

```diff
@@ -0,0 +1,315 @@
+- https://github.com/rust-lang/rust
+
+The swift organization migrated in https://forums.swift.org/t/swift-bugs-are-moving-to-github-issues-and-we-need-your-help/56125
+
+The key reasons to migrate away are:
+
+=== Developer and user experience
+
+GitHub conveys a lot more information about the status of issues, pull requests and releases than Jira does.
+When an issue or pull request link is pasted you can see whether it is open/closed or merged without having to open it.
+On clicking a commit you can see all the releases it appears on.
+
+When a PR is merged in GitHub it can automatically close an issue.
+
+A developer is already in GitHub and doesn't have to load another system. Jira is manually maintained in respect of:
+- status
+- links to work
+- released in
+The data quality in Jira is lower than in GitHub due to it not getting as updated as often as GitHub.
+
+=== Jira server end of life February 2024
+
+Atlassian announced the link:https://www.atlassian.com/migration/assess/journey-to-cloud[end of life of Jira server] in February of 2021.
+Open source projects may continue being sponsored by creating a link:https://www.atlassian.com/software/views/open-source-license-request[Jira cloud request].
+
+Jira cloud has link:https://support.atlassian.com/jira-cloud-administration/docs/explore-jira-cloud-plans/[limitations] that would affect us around:
+
+- user limits
+- authentication
+
+=== Mobile support
+
+Jira's mobile support is very poor, commenting doesn't work on mobile web and some fields are hidden and you can't get to all the information. There is a mobile app which works reasonably well but you have to log in almost every time you open it which is not a good experience.
+
+GitHub's issue on mobile  work really well.
+
+=== Integration with other tooling
+
+Many tools and processes integrate well with GitHub that will allow us to onboard more new contributors and help tackle low hanging fruit.
+
+==== 'good first issue' / 'help wanted'
+
+GitHub highlights repositories that are good for contributors to get started on in a codebase.
+See the Node.js link:https://github.com/nodejs/node/contribute[contributing page on GitHub].
+
+Also see the blog post on this: link:https://github.blog/2020-01-22-browse-good-first-issues-to-start-contributing-to-open-source/[Browse good first issues to start contributing to open source].
+
+==== hacktoberfest
+
+Issues labelled `hacktoberfest` are labelled by the Hacktoberfest project and will help new contributors find issues that they will like to work on.
+
+==== GitHub CLI
+
+GitHub has a really powerful CLI that can be used to automate management, retrieve issues assigned to the user or with a certain label and update the issue.
+All without having to visit the browser at all.
+
+==== Issue bounties
+
+It would enable issue bounty systems that integrate with GitHub as requested in the link:https://groups.google.com/g/jenkinsci-dev/c/Z2PLVBEGar0/m/pjlesVANBAAJ[mailing list].
+
+== Reasoning
```

Comment:
Need to go into the objections enumerated by @basil in https://groups.google.com/g/jenkinsci-dev/c/GUFCfBM1j_4/m/FbuslhB7AQAJ.